package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
-    "arsenal": "scality/Arsenal#879823c4",
+    "arsenal": "scality/Arsenal#feature/S3C-2002-admin-service",
```

Comment:
are we going back to referencing branches instead of commit hashes?

Reply:
No, this change crept in. I am going to add the commit hash

Reply:
But yes, for S3C we are going to do branches. I will have to do those changes after the release.